src/database/schema/ChannelListSchema.ts

```diff
@@ -448,7 +465,13 @@ class ChannelList implements BaseDbSchema {
       anon_user_info_color_code: json?.anon_user_info_color_code,
       anon_user_info_color_name: json?.anon_user_info_color_name,
       anon_user_info_emoji_name: json?.anon_user_info_emoji_name,
-      anon_user_info_emoji_code: json?.anon_user_info_emoji_code
+      anon_user_info_emoji_code: json?.anon_user_info_emoji_code,
+      isLastUpdatedByMe: json?.is_me,
+      firstMessage: {
+        message: json?.message || null,
+        type: json?.type,
+        attachmentJson: json?.chat_attachment_json
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fromJSON` method has been updated to handle the new `firstMessage` and `isLastUpdatedByMe` properties. It looks good, but please ensure that the `json` object always contains the `is_me`, `message`, `type`, and `chat_attachment_json` properties, or else it could lead to unexpected behavior.


<!-- This is an auto-generated comment by OSS CodeRabbit -->